src/DotnetDocument/DotnetDocument.csproj

```diff
@@ -1,11 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+    <PropertyGroup>
+      <TargetFramework>net8.0</TargetFramework>
+    </PropertyGroup>
```

Comment:
Why not updating the centralized version set in Directory.Build.props? Currently it is still set to net6.0

Reply:
Honestly, I was not expecting anything to happen on Github. I wanted to just get a working version out quickly and contribute back what I changed.

Reply:
Appreciate that and would be happy for these changes to get merged into main.
It's easier and cleaner to just just remove these project level overrides and update SDK once in the centralized version definition [here](https://github.com/pknobloch/dotnet-document/blob/7c22b049ceddacc9c5cec8824d8788d1bcd93e86/Directory.Build.props#L10).

Reply:
Directory.Build.props updated.

Reply:
Could you please also undo adding `<TargetFramework>` in `csproj`s? Otherwise they would take precedence over the centralized version.